src/Components/Chips/Chips.jsx

```diff
@@ -0,0 +1,68 @@
+        </label>
+        <label className={styles.Button.text}  onClick={() => filterDispatch({ type: "CATEGORY", payload: "Coding" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton} />Coding
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Talks" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Talks
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Finanace" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Finanace
+        </label>
+         <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Mystery" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Dhruv Rathee
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "StockMarket" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Stock Market
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Cyrpto" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Cyrpto
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Bollywood" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Bollywood
+        </label>
+          <label className={styles.Button.text}  onClick={() => filterDispatch({ type: "CATEGORY", payload: "Coding" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton} />Apna Collage
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Spirituality" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Spirituality
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Mystery" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Mystery
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "StockMarket" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Sandeep Maheshwari
+        </label>
+        <label className={styles.Button.text} onClick={() => filterDispatch({ type: "CATEGORY", payload: "Talks" })}>
+            <input type="radio" name="myRadio" className={styles.Button.radioButton}/>Famous CEOs
+        </label>
+    </div> 
+       )
```

Comment:
this is not right way yo render chips, make an array of chips placed in statics folder and use map function to render